django/researchdata/models.py

```diff
@@ -172,6 +173,18 @@ class Entity(models.Model):
         validators=[MinValueValidator(1), MaxValueValidator(31)],
         verbose_name='date (day)'
     )
+    year_range_from = models.IntegerField(
+        blank=True,
+        null=True,
+        validators=[MinValueValidator(1900), MaxValueValidator(2030)],
```

Comment:
Should the ranges for the `range_from` be earlier than those for `range_to`? This is probably dependent on the context, which I don't know. However, in general one might expect them to be different.

Reply:
yeah if supplied they should be but will often be blank/null for one or the other so they're happy to not enforce this